### PR TITLE
small fixing in patient invoice page and translation

### LINKED ISSUE #820: Providing a default description in patient invoice
Our patient invoice User interface does not let you submit the bill if the description is not provided, so it can be boring for a user to provide a description every time, essentially if the hospital can records a big amount of record per day.
So the user can be lazy and provide a bad description just to be able to post their invoice, so I think a default description is worth.
@IMA-WorldHealth/bhima-core what do you think?

### PR DESCRIPTION
**Pull request by DedrickEnc**
This PR fixes #820 , #971 , #970 
Thank you for contributing!

Before submitting this pull request, please verify that you have:
 - [x] Run your code through JSHint.  [Check out our styleguide](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/docs/STYLEGUIDE.md).
 - [x] Run integration tests.
 - [x] Run end-to-end tests.
 - [x] Accurately described the changes your are making in this pull request.

For a more detailed checklist, [see the official review checklist](https://docs.google.com/document/d/1nupLVLRXgSZJQo_acLgrwvPnN8RukfSiwRhSToj81uU/pub) that this PR will be evaluated against.

Ensuring that the above checkboxes are completed will help speed the review process and help build a stronger application.  Thanks!

### CODE REVIEW

File: client/src/partials/patient_invoice/patientInvoice.js [hunk 1]
```diff
@@ -157,6 +162,17 @@ function PatientInvoiceController(Patients, PatientInvoices, PatientInvoiceForm,
     }
   }
 
+  function handleServiceChange () {
```
Comment (sfount) [inline client/src/partials/patient_invoice/patientInvoice.js:165]: :+1: this is clear.

File: client/src/partials/patient_invoice/patientInvoice.js [hunk 2]
```diff
@@ -157,6 +162,17 @@ function PatientInvoiceController(Patients, PatientInvoices, PatientInvoiceForm,
...
+    vm.Invoice.setService(vm.Invoice.service);
+    updateDescription();
```
Comment (sfount) [inline client/src/partials/patient_invoice/patientInvoice.js:167]: Significantly reduced dependencies, requests and lines of code. Great work for maintainability.

**Review by sfount — COMMENTED** [review]
Code here LGTM! Confirmed that this addresses comments from the previous review.

File: client/src/partials/patient_invoice/patientInvoice.html [hunk 3]
```diff
@@ -35,8 +35,9 @@
                   <select
                     class="form-control"
                     name="service"
-                    ng-model="PatientInvoiceCtrl.Invoice.details.service_id"
-                    ng-options="service.id as service.name for service in PatientInvoiceCtrl.services"
+                    ng-model="PatientInvoiceCtrl.Invoice.service"
```
Comment (sfount) [inline client/src/partials/patient_invoice/patientInvoice.html:38]: The end to end tests use the model `service_id` to locate this component. They will need to be updated to pass with this model/ value change.

**Comment by jniles** [comment]
Just a note - because this no longer uses the `service_id` as the selector, it doesn't play well with `AppCache`.  Services are no longer recoverable from cache.

Steps to reproduce:
 1. Create a bill for TPA1.
 2. Select *distributable*, *11/01/2016*, and *test service* as details.
 3. Make an invalid row (for example, with a negative cost)
 4. Submit.
 5. Refresh the browser.
 6. Select TPA1.
 7. Click "Recover Cache"

You will notice that *distributable* is set, *11/01/2016* is the date, and the old description is back.  But *test service* is not set.

We can fix this in a separate PR if necessary.

**Comment by DedrickEnc** [comment]
Merci, I Weill see that tomorow

**Comment by sfount** [comment]
This looks good to me - this pull request addresses the comments and review made in https://github.com/IMA-WorldHealth/bhima-2.X/pull/985. 

@jniles has pointed out that this breaks the cache, if a cached invoice is loaded the service in the description will be wrong. This feature can be made use of right now - @DedrickEnc Can you create an issue to track this bug?

**Pull request merged**